react/features/base/connection/actions.web.js

```diff
@@ -8,6 +8,11 @@ import './reducer';
 const JitsiConferenceEvents = JitsiMeetJS.events.conference;
 const logger = require('jitsi-meet-logger').getLogger(__filename);
 
+export {
+    connectionEstablished,
```

Comment:
I don't like actions of a base, non-UI feature should be split between .web and .native. Temporarily though while we move more of the logic of the current Web into React, Illia chose to split them. I'll try very hard to remove the split here (and in functions.js where we have them currently).